### PR TITLE
update default radius, updatestats, added astral proxy

### PR DESCRIPTION
**Pull request by anjeshshrestha**

### CODE REVIEW

File: src/main/java/com/github/manolo8/darkbot/config/Config.java [hunk 1]
```diff
@@ -186,7 +186,7 @@ public static class Miscellaneous {
         public @Option @Number(max = 60 * 12, step = 10) int REFRESH_TIME = 60;
         public @Option @Number(max = 60 * 12, step = 10) int PAUSE_FOR = 0;
         public @Option boolean RESET_REFRESH = true;
-        public @Option @Visibility(Level.INTERMEDIATE) boolean UPDATE_STATS_WHILE_PAUSED = true;
+        public @Option @Visibility(Level.INTERMEDIATE) boolean UPDATE_STATS_WHILE_PAUSED = false;
```
Comment (Pablete1234) [inline src/main/java/com/github/manolo8/darkbot/config/Config.java:189]: This is working as intended, keep it as-is

File: src/main/java/com/github/manolo8/darkbot/core/objects/facades/AstralSelectionProxy.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.github.manolo8.darkbot.core.objects.facades;
+
+import com.github.manolo8.darkbot.core.itf.Updatable;
+
+public class AstralSelectionProxy extends Updatable {
+    @Override
+    public void update() {
+
+    }
```
Comment (Pablete1234) [inline src/main/java/com/github/manolo8/darkbot/core/objects/facades/AstralSelectionProxy.java:9]: A proxy with no code isn't terribly useful, i'd just delete this class unless it's s till WIP and you plan on finishing it within this PR

File: src/main/java/com/github/manolo8/darkbot/core/manager/GuiManager.java [hunk 3]
```diff
@@ -48,19 +48,24 @@ public class GuiManager implements Manager, GameScreenAPI {
...
     public final GroupManager group;
 
     private LoadStatus checks = LoadStatus.WAITING;
     private enum LoadStatus {
-        WAITING(gm -> gm.quests.lastUpdatedOver(5000) && gm.quests.visible),
-        MISSION_CLOSING(gm -> gm.quests.show(false)),
+        WAITING(gm -> (gm.quests.lastUpdatedOver(5000) && gm.quests.visible)
+                || (gm.monthlyDeluxe.lastUpdatedOver(5000) && gm.monthlyDeluxe.visible)),
+        MISSION_CLOSING(gm -> gm.quests.visible && gm.quests.show(false)),
+        MONTHLY_CLOSING(gm -> gm.monthlyDeluxe.visible && gm.monthlyDeluxe.show(false)),
```
Comment (Pablete1234) [inline src/main/java/com/github/manolo8/darkbot/core/manager/GuiManager.java:68]: Quests may not be visible at all (if you do not have an active running quest, it won't open).
You should not be waiting until they're visible to pass thru the state and allow it to move forward to click ammo once.

**Review by Pablete1234 — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/manolo8/darkbot/core/manager/GuiManager.java (hunk 3 above)
Comment (anjeshshrestha) [reply src/main/java/com/github/manolo8/darkbot/core/manager/GuiManager.java:68]: ahhh I didn't know it gets stuck 😓

File: src/main/java/com/github/manolo8/darkbot/core/manager/GuiManager.java [hunk 4]
```diff
@@ -48,19 +48,24 @@ public class GuiManager implements Manager, GameScreenAPI {
     public final Gui quests;
     public final Gui minimap;
     public final Gui targetedOffers;
+    public final Gui monthlyDeluxe;
```
Comment (Pablete1234) [inline src/main/java/com/github/manolo8/darkbot/core/manager/GuiManager.java:51]:
```suggestion
```

File: src/main/java/com/github/manolo8/darkbot/core/manager/GuiManager.java [hunk 5]
```diff
@@ -48,19 +48,24 @@ public class GuiManager implements Manager, GameScreenAPI {
...
     public final OreTradeGui oreTrade;
     public final GroupManager group;
 
     private LoadStatus checks = LoadStatus.WAITING;
     private enum LoadStatus {
-        WAITING(gm -> gm.quests.lastUpdatedOver(5000) && gm.quests.visible),
+        WAITING(gm -> (gm.quests.lastUpdatedOver(5000) && gm.quests.visible)
+                || (gm.monthlyDeluxe.lastUpdatedOver(5000) && gm.monthlyDeluxe.visible)),
         MISSION_CLOSING(gm -> gm.quests.show(false)),
+        MONTHLY_CLOSING(gm -> gm.monthlyDeluxe.show(false)),
```
Comment (Pablete1234) [inline src/main/java/com/github/manolo8/darkbot/core/manager/GuiManager.java:68]:
```suggestion
        WAITING(gm -> gm.quests.lastUpdatedOver(5000) && gm.quests.visible),
        MISSION_CLOSING(gm -> gm.quests.show(false)),
```

File: src/main/java/com/github/manolo8/darkbot/core/manager/GuiManager.java [hunk 6]
```diff
@@ -97,10 +102,13 @@ public GuiManager(Main main, PluginAPI pluginAPI, RepairManager repairManager) {
         this.quests = register("quests");
         this.minimap = register("minimap");
         this.targetedOffers = register("targetedOffers", TargetedOfferGui.class);
+        this.monthlyDeluxe = register("monthly_deluxe");
```
Comment (Pablete1234) [inline src/main/java/com/github/manolo8/darkbot/core/manager/GuiManager.java:105]:
```suggestion
```

**Review by Pablete1234 — CHANGES_REQUESTED** [review]
Let's make monthly deluxe closing part of a separate PR.
I'm certain it needs more work to handle the situation better and it cannot be just embedded in the current load sequence, because honestly, the current load sequence is already broken (eg: if you do not have a mission it will not go thru). So it needs to be checked separately.

**Review by Pablete1234 — APPROVED** [review]

**Pull request merged**